.github/workflows/windows-syscollector-tests.yml

```diff
@@ -0,0 +1,114 @@
+name: Syscollector test on Windows
+
+on:
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Maybe we should stay with a fixed version instead of latest (like macOS) to prevent a breaking change to make the tests fail.